### PR TITLE
adding insults

### PR DESCRIPTION
**Pull request by sophiedeziel**

### CODE REVIEW

**Comment by benichu** [comment]
Wat?

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# Insults
+
+{ insert insults here }
```
Comment (sophiedeziel) [inline README.md]: Change the curly brackets for square brackets

Comment (sophiedeziel) [inline README.md]: No.

Comment (garyharan) [inline README.md]: you_win_prize_downs

**Pull request closed without merging**